source/_components/aruba.markdown

```diff
@@ -18,28 +18,34 @@ redirect_from:
 
 This platform allows you to detect presence by looking at connected devices to an [Aruba Instant](http://www.arubanetworks.com/products/networking/aruba-instant/) device.
 
+The integration supports both REST API (preferrered) and SSH collection methods.  In order to enable the REST method, you must enable it on your access point:
+
+```
+(Instant AP)# conf t
+(Instant AP)(config)# allow-rest-api
+(Instant AP)(config)# end
+(Instant AP)# commit-apply
+```
+
```

Comment:
Let's move this into its own separate paragraph instead of stuffing it into the introduction text.

Additional 2 small textual suggestions changes:

````suggestion
The integration supports both REST API (preferred) and SSH collection methods. In order to enable the REST method, you must enable it on your access point:

```
(Instant AP)# conf t
(Instant AP)(config)# allow-rest-api
(Instant AP)(config)# end
(Instant AP)# commit-apply
```

````